lms/static/coffee/src/navigation.js

```diff
@@ -0,0 +1,70 @@
+var edx = edx || {},
```

Comment:
@andy-armstrong This is the file I had to replace because the pipeline was looking for it. Once we remove the requirement from the pipeline, we can remove this file.